test/hotspot/jtreg/compiler/loopopts/superword/TestUnorderedReduction.java

```diff
@@ -38,9 +62,22 @@ public class TestUnorderedReduction {
     static final int ITER  = 10;
 
     public static void main(String[] args) {
-        TestFramework.runWithFlags("-Xbatch",
-                                   "-XX:CompileCommand=compileonly,compiler.loopopts.superword.TestUnorderedReduction::test*",
-                                   "-XX:MaxVectorSize=16");
+        TestFramework framework = new TestFramework(TestUnorderedReduction.class);
+        framework.addFlags("-Xbatch",
+                           "-XX:CompileCommand=compileonly,compiler.loopopts.superword.TestUnorderedReduction::test*");
+
+        if (args.length != 1) {
+            throw new RuntimeException("Test requires exactly one argument!");
+        }
+
+        switch (args[0]) {
+            case "vanilla-U" -> { framework.addFlags("-XX:-AlignVector"); }
+            case "vanilla-A" -> { framework.addFlags("-XX:+AlignVector"); }
+            case "v016-U"    -> { framework.addFlags("-XX:MaxVectorSize=16"); }
+            case "v032-A"    -> { framework.addFlags("-XX:MaxVectorSize=32", "-XX:+AlignVector"); }
```

Comment:
Is it possible to use full words here and in tests ids? `v016-U` -> `VectorSize16-Unaligned`.

Reply:
Renamed them. Thanks for the suggestion @vnkozlov !